app/build.gradle

```diff
@@ -238,7 +238,13 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp:3.12.13"
 
     // Media player
-    implementation "com.google.android.exoplayer:exoplayer:${exoPlayerVersion}"
+    implementation "com.google.android.exoplayer:exoplayer-core:${exoPlayerVersion}"
+    implementation "com.google.android.exoplayer:exoplayer-dash:${exoPlayerVersion}"
+    implementation "com.google.android.exoplayer:exoplayer-database:${exoPlayerVersion}"
+    implementation "com.google.android.exoplayer:exoplayer-datasource:${exoPlayerVersion}"
+    implementation "com.google.android.exoplayer:exoplayer-hls:${exoPlayerVersion}"
+    implementation "com.google.android.exoplayer:exoplayer-smoothstreaming:${exoPlayerVersion}"
```

Comment:
Do we really use smoothstreaming inside NewPipe? 
AFAIK we only use DASH, HLS and progressive HTTP but I could be wrong...

Reply:
SmoothStreaming code is currently used in `PlaybackResolver` and `PlayerDataSource`, but after #8153, more code related to SmoothStreaming (support of URI and non-URI contents of this technology) will be added (even if no content with this adaptive streaming technology is currently extracted by the extractor).

Reply:
@TiA4f8R
Do we have anything inside the extractor that uses smooth streaming in the first place?

Reply:
@litetex No.

Reply:
Then I would remove support for SS in NewPipe until it actually becomes needed. It makes no sense to keep untested and unused code into the repo, which also increases app size by a little.

Reply:
That can be done in another PR if so.